packages/fast-components/src/slider-label/slider-label.template.ts

```diff
@@ -0,0 +1,28 @@
+import { html, when, ref } from "@microsoft/fast-element";
+import { SliderLabel } from "./slider-label";
+
+export const SliderLabelTemplate = html<SliderLabel>`
+    <div
+        ${ref("root")}
+        part="slider-label"
+        class="slider-label"
+        style=${x => x.positionStyle}
+    >
+        <div class="container">
+            ${when(
+                x => !x.hideMark,
+                html`
+                <div class="mark">
+                </div>    
+            `
+            )}
+            <div class="label-positioner">
+                <slot name="label">
+                    <span class="label"}>
+                        ${x => x.label}
+                    </span>
```

Comment:
Should we have the slot wrapping the span or the span wrapping the slot?

Reply:
I think this is why I have the extra dom you mentioned so I can control the positioning of whatever is slotted here, the span is the default content, it has no positioning, thus the thing wrapping the slot and its default content

Reply:
Actually, I think this is going to act like the default value for the slot. If I pass my own slotted content I think it'll take the place here.

Reply:
done, I believe as we discussed.